dune

```diff
@@ -39,3 +39,35 @@
 (alias
  (name runtest)
  (deps test-suite/summary.log))
+
+; Waiting for dune#1377
+; (install
+;  (section doc)
+;  (package coq-refman)
+;  (files
+;    (doc/sphinx/_build/html/index.html as refman/html/index.html)))
+
+; This will hopefully be available soon, see
+; https://github.com/ocaml/dune/issues/1377
+; (rule
+;   (targets doc/sphinx/_build/html/
+
+(alias
+ (name refman)
+ (deps
+  ; We could use finer dependencies here.
+  ; theories/Init/Prelude.vo
+  ; config/coq_config.py
+  ; tools/coqdoc/coqdoc.css
+  ; @rgrinberg mentions that this should be supported.
+  ; %{bin:coqdoc}
+  ; Must find a solution for the _static folder
+  ; doc/sphinx/_static/
+  (package coq)
+  (source_tree doc/sphinx)
+  (source_tree doc/tools))
+ (action (progn
+          ; This will force us to build coqdoc for now
+          (run coqdoc)
+          (run sphinx-build -j4 -b html -d doc/sphinx/_build/doctrees doc/sphinx doc/sphinx/_build/html))))
```

Comment:
Looking at this line, it gives the impression that the PDF format of the manual is not built. Can this be fixed easily? Can we get the same semantics for the refman targets than in `Makefile.doc` (with `refman` being the general target, and `refman-pdf` and `refman-html` being one format only)?

Reply:
Sure, we can, depending on the timeline we can use a more advanced solution or we can keep a bit of makefile for this purpose. Dune still don't supports well rules that generate a whole dir.